styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -359,19 +359,20 @@ void patchState(String componentId, WorkflowState state) throws IOException {
   }
 
   Optional<String> getDockerImage(WorkflowId workflowId) throws IOException {
-    final Key workflowKey = workflowKey(workflowId);
-    Optional<String> dockerImage = getOptStringProperty(datastore, workflowKey, PROPERTY_DOCKER_IMAGE);
+    Optional<String> dockerImage =
```

Comment:
Could this variable be named such that it reflects where the value comes from? Something like

```java
final Optional<String> workflowDefinitionDockerImage = ...
```